cfme/tests/cli/test_appliance_console.py

```diff
@@ -726,9 +726,8 @@ def test_appliance_console_ha_setup_dc():
     pass
 
 
-@pytest.mark.manual
 @pytest.mark.tier(2)
-def test_appliance_console_restart():
+def test_appliance_console_restart(appliance):
```

Comment:
I think this TC means restart appliance via appliance console
>> 18) Restart Appliance

We do not have any CLI option to restart the  EVM Server Processes

Reply:
@mnadeem92 you are right, We do not have restart command so performed stop and start operation as per steps mentioned in the testcase,

Reply:
@mnadeem92 As per our offline discussion I am going to implement this TC using option `18. Restart Appliance` to restart the appliace.